src/pal/src/misc/corefx.cpp

```diff
@@ -346,3 +348,68 @@ ForkAndExecProcess(
     return success ? 0 : -1;
 }
 
+void CopyStat(struct corefx_stat* dst, const struct stat* src)
+{
+    dst->mode = src->st_mode;
```

Comment:
Fixed.  BTW, I don't believe that CoreFX actually consumes this field given what we have today.  Would you prefer I drop it like I dropped some other fields?